include/TrustWalletCore/TWCoinType.h

```diff
@@ -38,6 +38,7 @@ enum TWCoinType {
     TWCoinTypeICON = 74,
     TWCoinTypeIOST = 291,
     TWCoinTypeIocoin = 295,
+    TWCoinTypeIoTeX = 304,
```

Comment:
IoTeX has index 304 and symbol IOTX, https://github.com/satoshilabs/slips/blob/master/slip-0044.md